packages/url_launcher/url_launcher_linux/lib/url_launcher_linux.dart

```diff
@@ -51,4 +51,16 @@ class UrlLauncherLinux extends UrlLauncherPlatform {
       },
     ).then((bool? value) => value ?? false);
   }
+
```

Comment:
I didn't add `launchUrl` to the desktop platforms in this PR because they don't currently support any mode switches, so it didn't seem as useful. (Someday I'll update them just so we can deprecate `launch`.)